.gitmodules

```diff
@@ -26,3 +26,6 @@
 [submodule "3rdparty/nvidia_cub"]
 	path = 3rdparty/nvidia_cub
 	url = https://github.com/NVlabs/cub.git
+[submodule "3rdparty/cutlass"]
```

Comment:
May I propose to have a dev@ discussion before adding a new dependency?

Reply:
Sure you can start one. On my side if there is no Cutlass then those will only work under certain dimensions.

Reply:
@marcoabreu Sure, let's start a dev discussion. Also, it would be really nice if we could later move the BatchedGEMM accelerations to the linalg_impl.h so other ops can be benefited : https://github.com/apache/incubator-mxnet/blob/15ea40d9bf7da3c4618ca45a6d023d9b0fb1c295/src/operator/linalg_impl.h#L325-L368
@ptrendx @eric-haibin-lin @szha

Reply:
Please feel free to start one yourself, thanks!

Reply:
Update: We decided to drop Cutlass from now, there are some cases on which Cublas is actually working better and it make the PR more simple